CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contributing
+
+### Supported Node.js versions
+
+All the current LTS versions are supported.
```

Comment:
Supports Node.js `>=14.0.0`. 

no?

Reply:
Why having to come here change the number when we can say that we support all 3 major version in LTS cycle :)